api/v01/entities/vrp_result.rb

```diff
@@ -44,6 +44,13 @@ class VrpResultSolutionRouteActivityDetails < Grape::Entity
       expose :timewindows, using: VrpResultSolutionRouteActivityDetailTimewindows, documentation: { is_array: true, desc: '' }
     end
 
+    class VRPResultDetailCosts < Grape::Entity
```

Comment:
I think instead of  `VRPResultDetailCosts` we can use one of the following. `DetailCosts` sounds wrong
```suggestion
    class VRPResultDetailedCosts < Grape::Entity
```
```suggestion
    class VRPResultCostDetails < Grape::Entity
```

Reply:
I like _VRPResultCostDetails_ more. It sounds more meaningfull for me.